news/2024/2024-05-01-the-nat-gazette-june-2024.md

```diff
@@ -0,0 +1,124 @@
+
+#### Progress bar for applications
+
+One of the issues with the opaqueness of the process was that candidates didn't know when their verdict would be delivered, or at what stage their application was. By offering our very own BN app parcel tracking service, applicants will now be able to be updated in real time when their food — I mean, their application — be delivered.
+
+Insert image perchance.
+
+#### Reworking application feedback
+
+Feedback has been simplified. In short, this change will make the feedback writing process which was deemed redundant be less time-consuming. This will quicken the time for the results of an application to be delivered. The evaluators' comments will instead serve as more in depth feedback.
+
+#### Communication platform post-applications (need a better title) (I couldn't come up with one either)
+
+After receiving their verdict, users can reply to their evaluators' feedback in the BN site. While the goal of this change is not to facilitate appeals as under normal circumstances the result of an application is unchangeable, applicants are encouraged to ask questions and clarification on their feedback by using this new feature.
+
+### Within the Beatmap Nominators
+
+#### Probation is no longer a demotion option
+
+The Probation Beatmap Nominators user group has been reworked slightly. It will now serve its original goal, which is as a trial ground for green beards! When a new Beatmap Nominator is allowed into the team, they will be put in probation for their first couple months. After evaluating their performance with actual BN work, they will either be promoted or removed. Full Beatmap Nominators cannot under any circumstance be demoted back into probation.
+
+#### Mock evaluations are no more
+
+Beatmap Nominators will not be able to make mock evaluations anymore. The system besides being underused, was usually superseded by the introduction of BN Evaluator cycles.
+
+#### Vetoes
+
+Vetoes have slightly been reworked. Once a Veto is made, the entire usergroup of the gamemode which it concerns will be drafted. While participation is not obligatory, this change aims to provide a better representation of what the BN group of a gamemode thinks of a Veto, in turn making the results feel less RNG depending on who got randomly chosen.
+
+#### Hybrid beatmapsets nomination changes
+
+Beatmapsets comprised of two or more rulesets should have an easier time getting Ranked! While the old system required two proficient BNs for each unique ruleset in the beatmapset, with the new changes only the main gamemode of a set will need two Beatmap Nominators. The rest of rulesets will only require one nomination from its respective mode. 
```

Comment:
```suggestion
Beatmap comprised of two or more game modes should have an easier time getting ranked! While the old system required two proficient BNs for each game mode present in a beatmap, with the new changes only the main game mode of a set will need two BNs. The other game modes will only require one nomination each. 
```

Reply:
Sounds a bit weird in the tongue leaving Beatmap without an article.

```suggestion
A beatmap comprised of two or more game modes should have an easier time getting ranked! While the old system required two proficient BNs for each game mode present in a beatmap, with the new changes only the main game mode of a set will need two BNs. The other game modes will only require one nomination each.
```

Reply:
believe i missed the plural